lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,138 +1,232 @@
+                        [disableRemoval]="saving"
+                        [categories]="categories"
+                        [managementMode]="categoriesManagementMode"
+                        [classifiableChanged]="classifiableChanged"
+                        (categoriesChange)="storeCategoriesToAssign($event)">
+                    </adf-categories-management>
+                </div>
+            </mat-expansion-panel>
         </ng-container>
+
         <mat-expansion-panel *ngFor="let customPanel of customPanels" [expanded]="canExpandTheCard(customPanel.panelTitle)">
             <mat-expansion-panel-header>
-                <mat-panel-title>{{ customPanel.panelTitle | translate }}</mat-panel-title>
+                <mat-panel-title class="adf-metadata-custom-panel-title">{{ customPanel.panelTitle | translate }}</mat-panel-title>
             </mat-expansion-panel-header>
             <adf-dynamic-component [id]="customPanel.component" [data]="{ node }"></adf-dynamic-component>
         </mat-expansion-panel>
-        <ng-container *ngIf="expanded">
-            <ng-container *ngIf="groupedProperties$ | async; else loading; let groupedProperties">
-                <div *ngFor="let group of groupedProperties; let first = first;"
-                     class="adf-metadata-grouped-properties-container">
-                    <mat-expansion-panel *ngIf="showGroup(group) || editable"
-                                         [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                                         [expanded]="canExpandTheCard(group.title) || !displayDefaultProperties && first">
-                        <mat-expansion-panel-header>
-                            <mat-panel-title>
-                                {{ group.title | translate }}
-                            </mat-panel-title>
-                        </mat-expansion-panel-header>
-
-                        <adf-card-view
-                            (keydown)="keyDown($event)"
-                            [properties]="group.properties"
-                            [editable]="editable"
-                            [displayEmpty]="displayEmpty"
-                            [copyToClipboardAction]="copyToClipboardAction"
-                            [useChipsForMultiValueProperty]="useChipsForMultiValueProperty"
-                            [multiValueSeparator]="multiValueSeparator"
-                            [displayLabelForChips]="true">
-                        </adf-card-view>
-                    </mat-expansion-panel>
 
-                </div>
-            </ng-container>
-            <ng-template #loading>
-                <mat-progress-bar mode="indeterminate" [attr.aria-label]="'DATA_LOADING' | translate">
-                </mat-progress-bar>
-            </ng-template>
+        <ng-container *ngIf="groupedProperties$ | async; else loading; let groupedProperties">
+            <div *ngFor="let group of groupedProperties; let first = first;"
```

Comment:
Can we apply *ngFor and class to mat-expansion-panel directly as this div contains only that element?

Reply:
you cannot mix `*ngIf` and `*ngFor` on the same element, Angular forbids that

Reply:
@DenysVuika you don't need to mix ngIf and ngFor. If you check exact structure of HTML then you will notice following: 

<img width="887" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/c97dd81f-a81b-43b0-b468-76d68d68d955">

So we can apply ngFor and styles to mat-expansion-panel directly and leave ngIf on ng-container. Thanks that we can remove div as it will become useless. But that is small comment, we can leave this structure

Reply:
Nevermind, I just noticed that mat-expansion-panel has own ngIf too... I'm closing this comment